js/src/events.ts

```diff
@@ -93,6 +93,9 @@ class EventModel extends WidgetModel {
             watched_events: [],
             ignore_modifier_key_events: false,
             prevent_default_action: false,
+            xy_trait_coords: null,
```

Comment:
I'm not sure `xy_trait_coords` is the most descriptive name for this. Maybe `xy_coordinate_system`?